types/leo-profanity/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for leo-profanity 1.5
+// Project: https://github.com/jojoee/leo-profanity (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Jack Humphries <https://github.com/jackhumphries9>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+export as namespace myLib;
```

Comment:
I dont see anywhere this is used in global script mentioned at https://www.npmjs.com/package/leo-profanity

Reply:
Yeah sorry I accidentally left that in from the generated boilerplate